x-pack/docs/en/rest-api/security/activate-user-profile.asciidoc

```diff
@@ -5,7 +5,10 @@
 <titleabbrev>Activate user profile</titleabbrev>
 ++++
 
-beta::[]
+NOTE: The user profile feature is designed for indirect use through {kib} and
+Elastic’s {observability}, {ents}, and {elastic-sec} solutions. Direct use is
+not supported. Elastic reserves the right to change or remove this feature in
+future releases without prior notice.
```

Comment:
```suggestion
NOTE: The user profile feature is designed only for use by {kib} and
Elastic’s {observability}, {ents}, and {elastic-sec} solutions. Individual
users and external applications should not call this API directly. Elastic reserves
the right to change or remove this feature in future releases without prior notice.
```

Reply:
@ywangd, here's a suggested change to the text that would apply to the notes in other files.